src/views/index.js

```diff
@@ -81,13 +85,23 @@ const template = (titlePrefix, ...elements) => {
       )
     );
 
+  const customLink = (filename) => {
```

Comment:
Wouldn't it be better if `customLink()` would keep the same parameter signature as `link()`, so in that sense it should accept an object like this example as an argument: `customLink({ rel:"stylesheet", href: "/custom-style.css"})`. Otherwise I would rename the function to `customCSS()`.

Reply:
Yes that makes sense since a link can be used for other objects.  This is a specific case for a custom CSS file, I don't see this being used for other links links icons.